barebox/env/boot/try-again

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+reset
```

Comment:
just asking to be sure: this behaviour is really what we want, e.g. infinite reset/boot loops when the board can't boot?

Reply:
If we don't do that, and the second partition update is broken, barebox would drop to its command line instead of rebooting to the working partition.